src/components/Card.tsx

```diff
@@ -10,6 +11,11 @@ export interface IProps {
   className?: string
 }
 
+const Container = styled.div`
+  h3 {
+    min-height: 67px !important;
```

Comment:
We are deprecating the usage of `emotion` in favor of using Chakra components. So, lets do this in a Chakra way. Check my next comment.